### ISSUE TITLE
user configuration is ignored (only applied to preview)

### ISSUE THREAD

**Issue by githubuser181226** [henrypp/matrix#6]
hi!
first, thank you for this great app. I's the best matrix screensaver I've seen so far.
I'm having a bit of an issue, though.
it seems like the screensaver ignores any custom settings I enter in the settings window.

ie. if I set "color hue" to red (value: 172), and lower the speed to 1, both changes are reflected only in the preview screen. however, if the screensever is launched while computer is idling, it always show the characters in green and the animation is faster, as if it was using all the default settings.

I tested both 32 and 64-bit versions of matrix.scr inside:
a) Win7 SP1 x64 Pro - I installed it using "install.bat" supplied in the package
b) WinXP SP3 x86 Pro - install.bat script failed (command not found), so I simply copied matrix.scr into C:\WINDOWS\system32

the issue is present in both cases though. custom configuration is ignored.
how can I remedy this?
please advise.

edit: sorry, after a reboot of my Win7 installation, everything seems to be good now, that is my custom configuration is being applied as expected.
still, not on my WinXP machine. there any custom settings are saved and correctly restored upon reopening the settings window, but are applied only in the preview display, not when the screensaver turns on on idle.

edit2: ok, I've changed the default values like hue and speed in the source and recompiled the program. it's not exactly the optimal way to solve my issue, but I don't feel like searching for another way.

**Comment by henrypp** [comment]
Put `matrix.ini` into `%APPDATA%\Henry++\matrix`. It happens because main executable have no rights for write to system directory.

**Comment by githubuser181226** [comment]
> 
> 
> Put `matrix.ini` into `%APPDATA%\Henry++\matrix`. It happens because main executable have no rights for write to system directory.

Hi. thanks for the response.

I checked and this file is already in place and it's being updated in realtime whenever I change an option. also, upon reopening the settings window every value that is stored in matrix.ini is being restored, and the little preview display reflects all these changes. this is also true for the fullscreen preview mode.

BUT, if I leave the computer idle, it's only then when the screensaver starts with all the default settings.

this issue persists only on my old WinXP SP1 installation. the local user belongs to Administrators group, has all the permissions. no domain, no internet, just strictly local operation.

at this moment I suspect it's the OS itself that is at fault.
I just changed the options in the source files and recompiled the screensaver. it still behaves exactly the same, though, just with my own default settings, so I'm ok for now :)
thanks. again.
